sklearn/feature_extraction/text.py

```diff
@@ -1593,8 +1597,6 @@ class TfidfVectorizer(CountVectorizer):
         If a callable is passed it is used to extract the sequence of features
         out of the raw, unprocessed input.
 
-        .. versionchanged:: 0.21
```

Comment:
the versionchanged is with reference to the following content, which should be indented.